content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,312 @@
+  grouped by the name of the assigned priority level and the reason for
+  rejection.
+    * Reason may be `queue-full`, indicating that too many requests were already
+      queued (this includes the case where the PriorityLevelConfiguration is
+      configured to reject rather than queue excess requests), or `time-out`,
+      indicating that the request was still in the queue when its deadline
+      expired.
+
+* `apiserver_flowcontrol_current_inqueue_requests` gives the instantaneous total
+  number of queued (not executing) requests for each priority level.
+
+* `apiserver_flowcontrol_current_executing_requests` gives the instantaneous
+  total number of executing requests for each priority level.
+
+* `apiserver_flowcontrol_request_queue_length` gives a histogram of queue
+  lengths for the queues in each priority level.
+    {{< note >}}
+    An outlier value in a histogram here means it is likely that a single flow
+    (i.e., requests by one user or for one namespace, depending on
+    configuration) is flooding the API server, and being throttled. By contrast,
+    if one priority level's histogram shows that all queues for that priority
+    level are longer than those for other priority levels, it may be appropriate
+    to increase that PriorityLevelConfiguration's concurrency shares.
+    {{< /note >}}
+
+* `apiserver_flowcontrol_request_concurrency_limit` gives the computed
+  concurrency limit (based on the API server's total concurrency limit and PriorityLevelConfigurations'
+  concurrency shares) for each PriorityLevelConfiguration.
+
+* `apiserver_flowcontrol_request_wait_duration_seconds` gives a histogram of how
+  long requests spent queued, grouped by the FlowSchema that matched the
+  request, the PriorityLevel to which it was assigned, and whether or not the
+  request successfully executed.
+    {{< note >}}
+    Since each FlowSchema always assigns requests to a single
+    PriorityLevelConfiguration, you can add the histograms for all the
+    FlowSchemas for one priority level to get the effective histogram for
+    requests assigned to that priority level.
+    {{< /note }}
+
+* `apiserver_flowcontrol_request_execution_seconds` gives a histogram of how
+  long requests took to actually execute, grouped by the FlowSchema that matched the
+  request and the PriorityLevel to which it was assigned.
+    
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+This feature is still in alpha. Its design and future plans can be found in the [relevant
+KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
+Feature requests and suggestions for improvements may be made to the [API
+Machinery SIG](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+
+Current plans include flow control for long-running requests (watches) and
+improved observability features (particularly logging and tracing).
```

Comment:
[Avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future). This is because this kind of text often goes out of date.

Reply:
:+1: